acrobat/scripts/scripts.js

```diff
@@ -206,4 +199,15 @@ const miloLibs = setLibs(LIBS);
       window.dispatchEvent(imsIsReady);
     }
   }, 1000);
+
+  loadScript('/acrobat/scripts/bowser.js');
```

Comment:
have we considered whether we could live without bowser? UA sniffing should be very limited at this point. Perhaps we could hardcode a couple of rules for IE, Edge?